amigos/amigos/log.txt

```diff
@@ -0,0 +1 @@
+Dict([(u'Datetime', datetime.datetime(2019, 6, 14, 14, 1, 53)), (u'RecNbr', 179), (u'R6', 1086.823486328125), (u'TCDT', nan), (u'R10', 1086.2347412109375), (u'R2_5', 1085.1212158203125), (u'R6_5', 1086.333740234375), (u'R8_5', 1086.3167724609375), (u'Q', nan), (u'R40', 1085.6849365234375), (u'Batt_volt', 11.579564094543457), (u'R4_5', 1085.0435791015625), (u'Ptemp_C', 24.85491943359375), (u'DT', nan), (u'R20', 1085.9552001953125)])
```

Comment:
Included by accident?

Reply:
haha! It has already been deleted from my file system.